AUTHORS.txt

```diff
@@ -38,3 +38,5 @@ Demetra Adrahtas
 Dan McDonald
 Hud Wahab
 Nicola Corna
+Robert Eckelmann
```

Comment:
I don't think you need to modify this file since you are already in it in the master

Reply:
When I did the implemtation and added me also here, it was not clear yet when my other PR was being merged.
Will change this